File: Python/Course.py
class Course:
    __nama_matakuliah = ""
    __sks = ""
    __kode_mk = ""

    def __init__(self, nama_matakuliah="", sks="", kode_mk=""):
        self.__nama_matakuliah = ""
        self.__sks = ""
        self.__kode_mk = ""

    def set_nama_matakuliah(self, nama_matakuliah):
        self.__nama_matakuliah = nama_matakuliah

    def set_sks(self, sks):
        self.__sks = sks

    def set_kode_mk(self, kode_mk):
        self.__kode_mk = kode_mk

    def get_nama_matakuliah(self):
        return self.__nama_matakuliah

    def get_sks(self):
        return self.__sks

    def get_kode_mk(self):
        return self.__kode_mk

File: CPP/Course.cpp

class Course
{
private:
    string nama_matakuliah;
    string sks;
    string kode_mk;

public:
    Course()
    {
        this->nama_matakuliah = "";
        this->sks = "";
        this->kode_mk = "";
    }
    Course(string nama_matakuliah, string sks, string kode_mk)
    {
        this->nama_matakuliah = nama_matakuliah;
        this->sks = sks;
        this->kode_mk = kode_mk;
    }
    void set_nama_matakuliah(string nama_matakuliah)
    {
        this->nama_matakuliah = nama_matakuliah;
    }
    void set_sks(string sks)
    {
        this->sks = sks;
    }
    void set_kode_mk(string kode_mk)
    {
        this->kode_mk = kode_mk;
    }
    string get_nama_matakuliah()
    {
        return this->nama_matakuliah;
    }
    string get_sks()
    {
        return this->sks;
    }
    string get_kode_mk()
    {
        return this->kode_mk;
    }
    ~Course() {}
};